checkstyle/suppressions.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE suppressions PUBLIC
-  "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
-  "https://checkstyle.org/dtds/suppressions_1_2.dtd">
-<suppressions>
-  <!-- Use @SuppressWarnings("...") if it is possible, only use this file if it is not -->
-
-  <suppress checks="LineLength"
-    files="BandcampExtractorHelper.java"
-    lines="54"/>
-
-  <suppress checks="LineLength"
-    files="ItagItem.java"
-    lines="19"/>
```

Comment:
Keep this file, just remove the lines I highlighted

Reply:
Why? Because of your comment above?